### ISSUE TITLE
Exemplos totalmente desatualizados, construtores sendo chamado de forma errada e etc.

### ISSUE THREAD

**Issue by marcinho1994** [pagarme/pagarme-php-sdk#24]
Os exemplos estão errados e desatualizados.

Como por um exemplo esse: https://github.com/pagarme/pagarme-php-sdk/blob/20b528c705c5f92b773e80ae4a59f281fa1fcd69/examples/orders/CreateOrderCreditCard.php

A classe o método construtor da classe (CreateOrderRequest) está com os argumentos diferentes do exemplo.
Além de que o (CreateCardRequest) não tem mais construtor, agora tem que atribuir os valores através de get/set.

Precisa de atualização os exemplos da SDK.

**Comment by walissonhms** [comment]
Verdade, notei que os exemplos estão desatualizados, o próprio exemplo de gerar boleto está com erro.

**Comment by vinhais** [comment]
É por conta desse SDK preguiçoso que foi feito com IA usando o [APIMATIC](https://www.apimatic.io/)

**Comment by PedroGuilhermeFariaDuarte** [comment]
meuuu..... a abstração desse SDK é até interessante mas o flow de uso é muito ruim.....  na própria documentação nem é usado esse sdk, é feito uma request com GuzzleHttp/CURL
